compiler/cpp/src/thrift/generate/t_erl_generator.cc

```diff
@@ -968,10 +969,13 @@ void t_erl_generator::export_string(string name, int num) {
 }
 
 void t_erl_generator::export_types_function(t_function* tfunction, string prefix) {
-
+  t_struct::members_type::size_type num = tfunction->get_arglist()->get_members().size();
+  if (num > std::numeric_limits<int>().max()) {
+    throw "integer overflow in t_erl_generator::export_types_function, name " + tfunction->get_name();
```

Comment:
true... I've forgot about compiler "patterns" :) yeah, we need to fix it globally